map/discovery_client_params.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "platform/preferred_languages.hpp"
+
+#include <string>
+#include <vector>
+
+namespace discovery
```

Comment:
How about to create a sub-directory, named "discovery" in the directory "map"?

Reply:
ok